docs/doc/30-reference/30-sql/00-ddl/10-database/ddl-drop-database.md

```diff
@@ -7,9 +7,11 @@ Drop a database.
 ## Syntax
 
 ```sql
-DROP DATABASE [IF EXISTS] <database_name>
+DROP { DATABASE | SCHEMA } [IF EXISTS] <database_name>
```

Comment:
I think we no need to add `CREATE SCHEMA` syntax, for some databases, schema is not same as database:
https://docs.snowflake.com/en/sql-reference/sql/create-schema.html

`SCHEMA`  keyword is only a compatibility, not a feature, let's keep it clear for the user.